### ISSUE TITLE
KeePassXD does not show the latest content of the database

### ISSUE THREAD

**Issue by dtgoitia** [Kunzisoft/KeePassDX#1518]
**Describe the bug**

KeePassXD has stopped showing latest versions of my `.kdbx` database.

**To Reproduce**

1. Edit `.kdbx` file - I do it on my laptop, and sync with my device via Syncthing.
2. Confirm the `.kdbx` file was updated. My file browser shows that the file _last updated_ timestamp was updated to the expected value. 
3. Open KeePassXD.
4. Select the `.kdbx` file.
5. The entries in the `.kdbx` file are out of date.

**Expected behavior**

5. The entries in the `.kdbx` file show latest updates.

**KeePass Database**

 - Created with: KeePassXC
 - Version: 2.7.4
 - Location: database downloaded to my device memory, is not a remote DB
 - File provider (`content://` URI): content://com.dropbox.product.android.dbapp.document_provider.documents/document/<long_hash>
 - Size: 1.33 MB
 - Contains attachment: Yes

**KeePassDX:**

 - Version: 3.5.0
 - Build: Free
 - Language: English

**Android:**

 - Device: Pixel 5
 - Version: 13

**Additional context**

Nothing to provide.

**Comment by J-Jamet** [comment]
KeePassDX has no cache and always displays the content of the data stream sent to it when the database is opened. When the database is opened a simple 'reload data' or 'merge data' is sufficient, but this is based on the principle that the file provider of the file manager works correctly, which is not the case for Dropbox.
The problem comes from your Cloud application, delete its cache and start the procedure again. Send a message to Dropbox so that they solve the problem or change the file manager application.

**Comment by J-Jamet** [comment]
https://github.com/Kunzisoft/KeePassDX/wiki/File-Manager-and-Sync#file-access-revocation

**Comment by J-Jamet** [comment]
Then, maybe you simply try not to open the right file, you indicate to use Syncthing but the content provider indicates Dropbox.

**Comment by dtgoitia** [comment]
I see, thanks for the clarification.

TL;DR: it was a human error, I have had a closer look and as you suggested
the issue was not related to KeePassXD.

Long version:
In the past I used Dropbox to sync my DB file. Some time ago I started
using Syncthing, and all worked as expected.

A few days back, I disconnected Dropbox, and I noticed that the file was
not syncing anymore. I thought I was opening the file synced via Syncthing,
but instead I was opening the file (which was) synced via Dropbox. The
confusion came from the fact that both files were under a very similar
directory structure and with identical file names.

It was a human error.

Thanks a lot for your help!
